### PR TITLE
[SRVKE-873] Update eventing dashboards

### PR DESCRIPTION
**Pull request by skonto**
a) add Event Count by Event Type (rate per minute) panel to broker-filter (triggers)
b) make panel order consistent for broker filter and ingress.
For example Event Count by Response Code Class (rate per minute) appears after failure rate for brokers.
c) source dashboard has a namespace variable that maps to a dropdown list which needs to be based on a different query. Current query is: https://github.com/openshift-knative/serverless-operator/blob/5d6160bc1d97a9a34cbef9b337bf48657241d37b/knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-source.yaml#L1175.
As not all sources have pods we should base this on the namespace query that matches the source namespace. This makes correlation much easier.
The right query is: "query": "label_values((pingsource_event_count{} OR kafkasource_event_count{} OR apiserversource_event_count{}), namespace_name)"
Also change namespace to EventNamespace to make the distinction clear.

Screenshots of the new dashboards:
image
image
image

### CODE REVIEW

**Comment by skonto** [comment]
May have to update due to https://github.com/openshift-knative/serverless-operator/pull/1043#issuecomment-865898532

**Comment by skonto** [comment]
Updated the `rate per minute` text to be more clear as discussed above:

image

@aliok pls review and approve, gentle ping.

**Comment by skonto** [comment]
@aliok gentle ping for approval.

**Review by aliok — APPROVED** [review]
/lgtm
/approve

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml [hunk 1]
```diff
@@ -215,7 +215,7 @@ data:
           "thresholds": "",
           "timeFrom": null,
           "timeShift": null,
-          "title": "Broker (broker-ingress): Success Rate  (2xx Event, rate per minute)",
+          "title": "Broker (broker-ingress): Success Rate  (2xx Event, fraction rate, over 1m window)",
```
Comment (rh-max) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml:218]: @skonto Note the double spacing between "Success Rate" and "(2xx..." - perhaps it wasn't intentional.

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml [hunk 2]
```diff
@@ -794,7 +794,7 @@ data:
           "thresholds": "",
           "timeFrom": null,
           "timeShift": null,
-          "title": "Trigger (broker-filter): Success Rate  (2xx Event, fraction rate per minute)",
+          "title": "Trigger (broker-filter): Success Rate  (2xx Event, fraction rate, over 1m window)",
```
Comment (rh-max) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml:797]: @skonto Another instance of double spacing between "Success Rate" and "(2xx...".

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-source.yaml [hunk 3]
```diff
@@ -589,7 +589,7 @@ data:
           "thresholds": "",
           "timeFrom": null,
           "timeShift": null,
-          "title": "MT PingSource: Success Rate  (2xx Event, fraction rate per minute)",
+          "title": "MT PingSource: Success Rate  (2xx Event, fraction rate, over 1m window)",
```
Comment (rh-max) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-source.yaml:592]: @skonto The last instance of double spacing between "Success Rate" and "(2xx...".

**Comment by skonto** [comment]
Thanks @rh-max created a PR here: https://github.com/openshift-knative/serverless-operator/pull/1220

**Pull request merged**